### DISCUSSION TITLE
git submodule update时报错

### DISCUSSION THREAD

**Discussion by 123de234** [hugo-next/hugo-theme-next#59] [Q&A] [answered]
<img width="311" alt="屏幕截图 2022-10-01 045650" src="https://user-images.githubusercontent.com/23733252/193354949-6b129220-26fc-4468-a0e3-e6dcc10c8b94.png">

如图所示

**Comment by 123de234** [comment, marked as answer]
解决了：

在 .gitmodule 中加 branch = main
then, go run 
git submodule sync
git submodule update --remote

It works eventually.

**Comment by elkan1788** [comment]
一般不需要执行 merge 行为， 是否你这边对主题做些自定义的改造哟？ 可以考虑直接在自己站点下添加 layouts 文件夹，在里面直接使用同名文件进行改造。
